shared/actions/config/index.tsx

```diff
@@ -212,11 +217,11 @@ function* loadDaemonAccounts(
       )
     }
 
-    const result = yield* Saga.callPromise(RPCTypes.configGetAllProvisionedUsernamesRpcPromise)
-    let usernames = result.provisionedUsernames || []
-    let defaultUsername = result.defaultUsername
-    usernames = usernames.sort()
-    const loadedAction = ConfigGen.createSetAccounts({defaultUsername, usernames})
+    const configuredAccounts: Array<RPCTypes.ConfiguredAccount> = yield* Saga.callPromise(
+      RPCTypes.loginGetConfiguredAccountsRpcPromise
```

Comment:
do we need to do this on login? maybe we can skip if its the first one? i want to avoid extra calls on start that we can avoid